### PR TITLE
https://issues.redhat.com/browse/ACM-7380 Remove some import steps

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @KevinFCormier. Here is the updated version. Does it look ok? Thanks!

**Review by KevinFCormier — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]
This looks good to me. Thanks Chris!

**Comment by swopebe** [comment]
Please do not merge this, needs some editing. TY

File: clusters/cluster_lifecycle/import_gui.adoc [hunk 1]
```diff
@@ -84,56 +84,38 @@ To configure your managed cluster klusterlet to run on specific nodes, see xref:
 
 This method requires you to run the provided command on the target cluster to import the cluster.
 
-*Important:* The command contains pull secret information that is copied to each of the imported clusters. Anyone who can access the imported clusters can also view the pull secret information. Consider creating a secondary pull secret at link:https://cloud.redhat.com/[Red Hat Hybrid Cloud] or create a service account to protect your personal credentials. 
+Provide the information in the console after selecting *Cluster* > *Import cluster*. 
 
-. Generate and copy the import command:
+For the {ocp-short} Dedicated environment only, complete the following steps:
```
Comment (swopebe) [inline clusters/cluster_lifecycle/import_gui.adoc]: @cadawson are we saying this whole section, manual steps, is for OCP dedicated only and if so, that really should be indicated towards the top, as it is an interruption to read after the initial step.

File: clusters/cluster_lifecycle/import_gui.adoc [hunk 2]
```diff
@@ -84,56 +84,38 @@ To configure your managed cluster klusterlet to run on specific nodes, see xref:
...
-. For the {ocp-short} Dedicated environment only, complete the following steps (if you are not using a {ocp-short} Dedicated environment, skip to step 3):
-
-.. Log in to the {ocp-short} console of the cluster that you want to import.
-
-.. Create the `open-cluster-management-agent` and `open-cluster-management` namespaces or projects on the cluster that you are importing.
-
-.. Find the klusterlet operator in the {ocp-short} catalog. 
-
-.. Install it in the `open-cluster-management` namespace or project that you created. 
+. Install it in the `open-cluster-management` namespace or project that you created.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/import_gui.adoc]: Per style manuals, we want to try to eliminate the "it" and "this" in our content and be more specific. 

Is the name of the operator simply "klusterlet" (what does the user see) @cadawson ?

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/cluster_lifecycle/import_gui.adoc]: In addition to the standard running of the command, you have to do the steps listed after only for OCP Dedicated.

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 2 above)
Comment (cadawson) [reply clusters/cluster_lifecycle/import_gui.adoc]: "Klusterlet operator" is the name of the operator in this context, @swopebe .

**Comment by dockerymick** [comment]
@swopebe thanks for agreeing to take this issue. are you comfortable with merging this PR?

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/cluster_lifecycle/import_gui.adoc]: I don't think that is answering the question, just quite yet bc to me it seems that whole section is for OCP Dedicated. (?) If that is the case, we should say that earlier. Other than that small change, or editing to make it clearer what it applies to, I think this is done.

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 2 above)
Comment (swopebe) [reply clusters/cluster_lifecycle/import_gui.adoc]: Yes, so we should say that.

**Review by swopebe — CHANGES_REQUESTED** [review]
small comment to address before merge

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/cluster_lifecycle/import_gui.adoc]: DM about this just now.

Comment (cadawson) [reply clusters/cluster_lifecycle/import_gui.adoc]: Ok, I think the updates address the issue. I made it apply only to the dedicated environment. Thanks @swopebe .

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 2 above)
Comment (cadawson) [reply clusters/cluster_lifecycle/import_gui.adoc]: I read too much into that one. Thanks.

File: clusters/cluster_lifecycle/import_gui.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/cluster_lifecycle/import_gui.adoc]: Oh, Ok, I was just going to move the content up higher but I think your solution works as well, I just wanted to make sure that is what you meant first.

**Review by swopebe — APPROVED** [review]
thanks for the edits.

**Pull request merged**